mergin/client.py

```diff
@@ -321,12 +321,26 @@ def get_push_changes(self):
                     }
                 else:
                     not_updated.append(file)
-            except (pygeodiff.GeoDiffLibError, pygeodiff.GeoDiffLibConflictError):
+            except (pygeodiff.GeoDiffLibError, pygeodiff.GeoDiffLibConflictError) as e:
                 pass  # we do force update
 
         changes['updated'] = [f for f in changes['updated'] if f not in not_updated]
         return changes
 
+    def get_list_of_push_changes(self, push_changes):
+        i = 0
+        changes = {}
+        for file in push_changes["updated"]:
+            if "diff" in file:
+                changeset_path = file["diff"]["path"]
+                changeset = self.fpath(changeset_path,  self.meta_dir)
+                result_file = self.fpath("change_list" + str(i), self.meta_dir)
+                self.geodiff.list_changes_summary(changeset, result_file)
```

Comment:
not sure if there is use case that changeset file will not be exist, or that list_summary will fail from other reason but maybe let's catch any geodiff exception